docs/security.md

```diff
@@ -0,0 +1,184 @@
+
+
+
+> :office: **GitHub Server**
+>
+> If you use the GitHub for Jira app with a GitHub Enterprise Server instance that you control yourself, you create your own GitHub app on that server instance which means that you can define which webhooks you want to send to the GitHub for Jira app.
+> 
+> If you don’t activate all webhooks you should expect that some features will not work.
+
+## Can I limit the access to my GitHub organization to certain IP addresses?
+
+Yes. The GitHub for Jira app operates from a [dedicated range of IP addresses](https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md). As long as these IP addresses are allowed access, you can restrict all other IP addresses.
+
+> :cloud: **GitHub Cloud**
+> 
+> By default, an organization in a GitHub Cloud account is accessible from any IP address. If you have and Enterprise account, you can make use of [GitHub’s IP allowlist feature](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-allowed-ip-addresses-for-your-organization) to restrict the range of IP addresses that have access to your GitHub organization.
+> 
+> Make sure to read through our [docs on IP allowlists](https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md).
+
+> :office: **GitHub Server**
+> 
+> If you use a self-hosted GitHub Server instance, you can restrict access to your GitHub server via any networking rules, as long as the [GitHub for Jira app’s IP addresses](https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md) are allowed to access the server.
+
+## What data is stored where? Is it encrypted?
+
+The GitHub for Jira app stores the following data in a database that is shared across multiple tenants.
+
+The data is encrypted at rest using AWS’s encryption feature.
+
+The database and the compute servers of the GitHub for Jira app are located in AWS’s us-west region.
+
+**GitHub Cloud + GitHub Server:**
+
+| Field | Description                                                                                                                                                  |
+| ----- |--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Jira host | The hostname of your Jira site (i.e. *.atlassian.net). This is used as an identifier for your Jira site.                                                     
+| Connect client key | The client key that is generated during installation of the app into your Jira site. This key is used to authorize all calls to your Jira site’s API.        |
+| Connect shared secret | The client secret that is generated during installation of the app into your Jira site. This secret is used to authorize all calls to your Jira site’s API.  |
```

Comment:
I think it is also encrypted with cryptor, right @gxueatlassian ?

Reply:
yes, it is, I looked it up in the code